distributed/http/worker/prometheus/core.py

```diff
@@ -34,20 +34,6 @@ def __init__(self, server: Worker):
     def collect(self) -> Iterator[Metric]:
```

Comment:
I gutted most of this class away; eventually it will be deleted completely.

Reply:
Are the metrics you've refactored here actually showing up in a prometheus scrape? Is some magic making that happen?

Reply:
Yes they're showing up.

The magic is that they are registered automatically on `prometheus_client.REGISTRY`.
In the final thing we'll actually have to use a local registry so that the scheduler doesn't publish the worker metrics and vice versa.